src/helper/Redux/service/reminders.service.js

```diff
@@ -1,36 +1,11 @@
-import { useContext, useState } from 'react';
+import { useContext } from 'react';
 import { AuthContext } from '../../AuthProvider/AuthProvider';
 import { db } from '../../Firebase/firebase';
 
-function Service() {
+export const GetAllReminders = async () => {
   const { currentUser } = useContext(AuthContext);
-  const userId = currentUser.uid;
-  const [data, setData] = useState([]);
-
-  const getAllReminders = () => {
-    db.collection('users')
-      .doc(userId)
-      .collection('reminders')
-      .onSnapshot((docs) => {
-        const currentState = [];
-        docs.forEach((doc) => {
-          currentState.push(doc.data());
-        });
-        setData(currentState);
-      })
-      .then(() => {
-        return data;
-      })
-      .catch((error) => {
-        console.log(error);
-      });
-  };
-
-  return {
-    getAllReminders,
-  };
-}
-
-const reminderService = Service();
-
-export default reminderService;
+  return await db
+    .collection('users')
+    .doc(currentUser.uid)
+    .collection('reminders');
+};
```

Comment:
here we will successfully return the object of collection. This is run perfectly no issues with this fn.